ratis-common/src/main/java/org/apache/ratis/protocol/exceptions/NotLeaderException.java

```diff
@@ -30,7 +30,7 @@ public class NotLeaderException extends RaftException {
   private final Collection<RaftPeer> peers;
 
   public NotLeaderException(RaftGroupMemberId memberId, RaftPeer suggestedLeader, Collection<RaftPeer> peers) {
-    super("Server " + memberId + " is not the leader" + (suggestedLeader != null? " " + suggestedLeader: ""));
+    super("Server " + memberId + " is not the leader" + (suggestedLeader != null ? " ,suggested leader is: " + suggestedLeader : ""));
```

Comment:
@weimingdiit , thanks for working on this!  The space should be after the comma, i.e.  `", suggested ..."`.

Reply:
@szetszwo Ok,i will update  it